src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -276,52 +272,75 @@
+                    parent_dir = os.path.dirname(output.workflow_output_file) if output.workflow_output_file else None
+
+                    await job_store.update_status(job_id, "success", output_path=str(parent_dir))
+            except Exception as e:
+                logger.exception("Error in evaluation job %s: %s", job_id, str(e))
+                await job_store.update_status(job_id, "failure", error=str(e))
+
+        async def start_evaluation(request: EvaluateRequest, http_request: Request):
             """Handle evaluation requests."""
 
             async with session_manager.session(http_connection=http_request):
+                assert self._job_store is not None and JobStatus is not None, \
```

Ruff (0.12.2)

Use of `assert` detected

(S101)